### DISCUSSION TITLE
Please add https://thepiratebay.org/ , https://rutor.info/ , https://rutracker.org/ , https://1337x.to/

### DISCUSSION THREAD

**Discussion by BlackHARD94** [iMAboud/iMSteam#2] [Ideas]
Please add https://thepiratebay.org/ , https://rutor.info/ , https://rutracker.org/ , https://1337x.to/

**Comment by batiyaa** [comment]
I agree with him

**Reply by BlackHARD94** [reply]
Батя дэв :D))

**Reply by batiyaa** [reply]
батия, а дев потому что был мал и глуп.

**Reply by batiyaa** [reply]
А всё, теперь просто батия. Заебок

**Reply by BlackHARD94** [reply]
ахах, всё равно прикольный ник

**Comment by BlackHARD94** [comment]
And It looks more authentic this way
![Безымянный](https://github.com/iMAboud/iMSteam/assets/14151135/b9f8c940-aa38-46c4-ae92-c5984e456bb0)

**Reply by iMAboud** [reply]
I did place them under "Add to Cart" at first but it didn't feel practical since you only use steam as a search engine to check the name + video and decide whether you want to look for it on other sites or not.. Scrolling down everytime doesn't feel like it!

And for the sites you suggested "Piratebay" is no longer supported, rutor, rutracker and 1337 have little to no moderation lately so it's easy to have a malware!
It's not very safe for the average user, however, I'm planning on adding a feature to add your own sites, or at least hide these sites by default and can be enabled later.

**Reply by BlackHARD94** [reply]
+++

**Comment by iMAboud** [comment]
I don't think adding every pirating site on the internet is ideal, but you can do this yourself with almost any site by following these steps using **Tampermonkey script** which I updated today: 

Go to any site e.g: https://rutracker.org/

Do a search, e.g: The Witcher

Notice the link changes to: https://rutracker.org/forum/tracker.php?nm=the+witcher

Remove "the+witcher" to be like this: https://rutracker.org/forum/tracker.php?nm=

Then add it to the user-script like this:

    const rutracker= `https://rutracker.org/forum/tracker.php?nm=${formattedGameName}`;
    createButton(rutracker, "RuTracker", "Search on RuTracker", "URL_FOR_ICON.PNG");

(Make sure to use this exact format, and add its name in lower case for IDs e.g (rutacker), also add a png icon link).

(Also note, if a site's search quarry is in the middle of the link just replace it with ${formattedGameName} )

e.g : normal link : https://1337x.to/sort-category-search/the%20witcher/Games/seeders/desc/1/
scripted : https://1337x.to/sort-category-search/${formattedGameName}/Games/seeders/desc/1/
